helper/logger.go

```diff
@@ -90,3 +94,17 @@ func FatalIf(err error, msg string, data ...interface{}) {
 	Logger.Println(5, "Stack trace: ", stackInfo())
 	os.Exit(1)
 }
+
+// To avoid looped dependency.
+// api.RequestIdFromContext() requires RequestContext, while RequestContext depends on meta package.
```

Comment:
Maybe split `RequestContext` into 3 and get `RequestId`, `BucketInfo`, `ObjectInfo` separately. Say, `RequestIdFromContext`, `BucketInfoFromContext` and `ObjectInfoFromContext`.

Reply:
RequestContext is defined in package api and depend on storage, while request ID is needed by other packages like storage, meta, ...
So just to avoid looped dependency, we define a new API in helper to handle request ID only.
*************************api/server.go***************************
const RequestContextKey = "RequestContext"

type RequestContext struct {
	RequestId  string
	BucketInfo *types.Bucket    //Depend on storage here.
	ObjectInfo *types.Object
}